### ISSUE TITLE
Tests fail with latest lxml 4.7.1 due to extra newlines

### ISSUE THREAD

**Issue by mauritsvanrees** [repoze/repoze.xmliter#8]
In issue #7 I said I would add a separate issue for this, but promptly forgot it.

The failures are in these three tests:

```
repoze.xmliter.tests.TestIterator.test_getHTMLSerializer_doctype_xhtml_serializes_to_xhtml
repoze.xmliter.tests.TestIterator.test_replace_doctype
repoze.xmliter.tests.TestIterator.test_replace_doctype_blank
```

Sample error, note the extra newlines at the end:

```
b'<!D[194 chars]>Hello, w&#246;rld!<img src="foo.png" /></body>\n</html>\n' != b'<!D[194 chars]>Hello, w&#246;rld!<img src="foo.png" /></body>\n</html>\n\n\n'

  File "/srv/python3.9/lib/python3.9/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/srv/python3.9/lib/python3.9/unittest/case.py", line 592, in run
    self._callTestMethod(testMethod)
  File "/srv/python3.9/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/home/jenkins/.buildout/eggs/cp39/repoze.xmliter-0.6-py3.9.egg/repoze/xmliter/tests.py", line 167, in test_replace_doctype
    self.assertEqual(
  File "/srv/python3.9/lib/python3.9/unittest/case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/srv/python3.9/lib/python3.9/unittest/case.py", line 830, in _baseAssertEqual
    raise self.failureException(msg)
```

See this [Plone Jenkins PR job](https://jenkins.plone.org/job/pull-request-6.0-3.9/732/#showFailuresLink).

Looks easy to fix at first glance, just strip the result, and I can make a PR. But let's get some reactions on the other more general issue first.

**Comment by tseaver** [comment]
@mauritsvanrees I would do the proposed fix ASAP, and push a `0.6.1` release.

**Comment by mauritsvanrees** [comment]
PR created: https://github.com/repoze/repoze.xmliter/pull/9
